stepactions/sealights/create-test-session/0.1/create-test-session.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: tekton.dev/v1beta1
+kind: StepAction
+metadata:
+  name: initialize-sealights-session
+  labels:
+    konflux-ci/sealights: "true"
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: konflux
+spec:
+  description: |
+    This StepAction initializes a Sealights test session by creating a new session ID
+    using the Sealights API and storing it for later use.
+  params:
+    - name: sealights-bsid
+      type: string
+      description: "The Sealights Build Session ID (BSID) associated with the build."
+    - name: test-stage
+      type: string
+      description: "The name or identifier of the testing phase (e.g., `integration`, `e2e`)."
```

Comment:
similarly like [in `upload-ginkgo-results` StepAction](https://github.com/konflux-ci/tekton-integration-catalog/pull/127/files#diff-4cf8eeed7fa1aaff64c4c9f1df8b8bcf7c0b840984ee8d1dfd601407e2ff84faR11-R14), should we add the param for `sealights-domain`?